src/assets/styles/adminPage.scss

```diff
@@ -0,0 +1,112 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    position: relative;
+    min-height: 100vh;
+  }
+
+.adminPageForm{
+    margin: 1rem 0 0 0;
+    padding: 0 5rem;
+}
+
+th{
```

Comment:
You should avoid using those HTML tags because it affects other components, you should remember that react is a one-page app.